.github/CONTRIBUTING.md

```diff
@@ -306,43 +306,68 @@ eg. on MacOS you can follow this instructions:
 #1 install brew (https://brew.sh)
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 
-#2 install node 16x using NVM (https://github.com/nvm-sh/nvm)
-nvm install v16.17.0`
-nvm alias default v16.17.0`
+# before next step uninstall node
+brew uninstall --ignore-dependencies node 
+brew uninstall --force node
+* delete .npm and .node folders from your root folder *
 
-#3 install postgres (https://wiki.postgresql.org/wiki/Homebrew)
+#2 install nvm
+brew update 
+brew install nvm
+
+# create a directory for nvm in your root folder
+mkdir ~/.nvm 
+
+# configure env variable
+vim ~/.bash_profile
+
+# paste these lines
+export NVM_DIR=~/.nvm
+source $(brew --prefix nvm)/nvm.sh
+
+# load the variable into your current env
+source ~/.bash_profile
+
+#3 install node 16x using NVM (https://github.com/nvm-sh/nvm)
+nvm install v16.17.0
+nvm alias default v16.17.0
+
+#4 install postgres (https://wiki.postgresql.org/wiki/Homebrew)
 brew install postgresql@14
 brew services start postgresql@14
 
-#4 install dbt (https://docs.getdbt.com/dbt-cli/install/homebrew)
-brew install dbt-labs/dbt/dbt-postgres@1.1.0
+#5 install dbt (https://docs.getdbt.com/dbt-cli/install/homebrew)
+brew install dbt-labs/dbt/dbt-postgres
 
-#5 install packages
+#6 install packages
 yarn
 
-#6 create `.env.local` and override any variables you need to change from `.env`
+#7 clone the repo and open it in your IDE
+git clone https://github.com/lightdash/lightdash.git
+
+#8 create `.env.local` and override any variables you need to change from `.env`
 touch .env.local
 open .env.local -t
 
 # here is a sample content of the `.env.local` file
 PGHOST=localhost
 PGPORT=5432
-PGUSER=pg_user
-PGPASSWORD=pg_password
+PGUSER= *USE YOUR MACHINE USERNAME*
+PGPASSWORD= *LEAVE BLANK*

```

Comment:
maybe add a note here. if it's a postgres version installed by the brew then PGUSER is machine user and PGPASSWORD is blank. but users might already have PG up and running with different credentials

Reply:
I agree, add a comment, saying which is the default for brew postgres, but do not "force", to avoid confusion.